tests/test_codebase/test_mmdet3d/test_mono_detection_model.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmengine
+import pytest
+import torch
+
+import mmdeploy.backend.onnxruntime as ort_apis
+from mmdeploy.codebase import import_codebase
+from mmdeploy.utils import Backend, Codebase
+from mmdeploy.utils.test import SwitchBackendWrapper, backend_checker
+
+try:
+    import_codebase(Codebase.MMDET3D)
+except ImportError:
+    pytest.skip(
+        f'{Codebase.MMDET3D} is not installed.', allow_module_level=True)
+from mmdeploy.codebase.mmdet3d.deploy.mono_detection_model import \
+    MonoDetectionModel
+
+nuscenes_pcd_path = 'tests/test_codebase/test_mmdet3d/data/nuscenes/n015-2018-07-24-11-22-45+0800.pkl'  # noqa: E501
```

Comment:
could use small image to speed-up testing

Reply:
I'm not familiar with this, do you have any suggestion image? I can just replace it.